### PR TITLE
JAMES-2154 implement move command for JPA mailbox

### PR DESCRIPTION
**Pull request by mschnitzler**
I implemented the move-method in JPAMessageMapper and wrote a unit test.

### CODE REVIEW

**Comment by mschnitzler** [comment]
issue #997

File: mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/JpaMessageMoveTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.apache.james.mailbox.jpa.mail;
```
Comment (chibenwa) [inline mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/JpaMessageMoveTest.java]: Sorry, you forgot the compulsory Apache 2.0 license that should be on top of each file...

File: mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/TransactionalMessageMapper.java [hunk 2]
```diff
@@ -132,7 +132,9 @@ public MessageMetaData copy(final Mailbox mailbox, final MailboxMessage original
 
     @Override
     public MessageMetaData move(Mailbox mailbox, MailboxMessage original) throws MailboxException {
-        return messageMapper.move(mailbox, original);
+        MessageMetaData data = messageMapper.execute(
```
Comment (chibenwa) [inline mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/TransactionalMessageMapper.java]: We could return directly?

Comment (mschnitzler) [reply mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/TransactionalMessageMapper.java]: I applied the same pattern as in the other methods (add, copy, etc.).
However, I fixed this (and the other methods) in commit [3700eab](https://github.com/linagora/james-project/pull/1850/commits/3700eabab49f92284ddc1b0f8297fc6d9e44a88c).

**Comment by chibenwa** [comment]
Hi,

This pull request has just been merged.

Thank you again for your dedication toward the James project. If you need help, want to [contribute](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20newbie%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC) [documentation](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20documentation%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), new [features](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20feature%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), tests or [bugfix](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20easyfix%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), please don't hesitate to reach us.

Thanks again,

Cheers,

Benoit

**Pull request closed without merging**